user_tools/src/spark_rapids_tools/cmdli/tools_cli.py

```diff
@@ -40,7 +40,7 @@ def qualification(self,
                       platform: str = None,
                       target_platform: str = None,
                       output_folder: str = None,
-                      filter_apps: str = None,
+                      filter_apps: str = QualFilterApp.tostring(QualFilterApp.get_default()),
```

Comment:
Same issue with `filter_apps` argument.
The filter_apps aregument has some complicated logic to get the default. It depends on the value of other arguments.

Reply:
Thanks @amahussein! Updated this.